api/messages/message.py

```diff
@@ -1,3 +1,7 @@
 class Message:
     def __init__(self, text):
         self.text = text
+        self.metadata = {
+            "api": "api_flask_python_hello-world",
+            "branch": "starter"
+        }
```

Comment:
I think it should be cleaner if we define its own class for the metadata property:
```py
class Message:
    def __init__(self, text):
        self.metadata = Metadata()
        self.text = text

class Metadata:
    def __init__(self):
        self.api = "api_flask_python_hello-world"
        self.branch = "starter"
```